python/py-postgresql/Portfile

```diff
@@ -5,31 +5,29 @@ PortGroup           python 1.0
 
 name                py-postgresql
 python.rootname     ${name}
-version             1.2.1
-revision            1
+version             1.3.0
+revision            0
 categories-append   devel databases
 license             BSD
 maintainers         nomaintainer
 description         Python interface to PostgreSQL
-long_description \
-   py-postgresql is a project dedicated to improving the Python interfaces \
-   to PostgreSQL. It strives to provide substantial convenience to Python \
-   programmers using PostgreSQL.  At its core, py-postgresql provides a \
-   PG-API, postgresql.api, and DB-API 2.0 interface for using a PostgreSQL \
-   database.
+long_description    ${name} is a project dedicated to improving the Python interfaces \
```

Comment:
Is this still narrower than 80 columns?

Reply:
I have no idea but I can make it narrower than it is.

Reply:
Thanks. Please keep portfiles (except for the modeline) to 80 columns.

Reply:
I made description narrower now.

Reply:
But it's still much wider than 80 columns. I'm not sure what the problem was with the old version of this, which was under 80 columns.

Reply:
I believe, description is not squeezed into 80 character-width.